mmdet3d/core/evaluation/seg_eval.py

```diff
@@ -66,28 +66,43 @@ def get_acc_cls(hist):
     return np.nanmean(np.diag(hist) / hist.sum(axis=1))
 
 
-def seg_eval(gt_labels, seg_preds, label2cat, logger=None):
-    """Semantic Segmentation Evaluation.
+def seg_eval(gt_labels,
+             seg_preds,
+             label2cat,
+             num_classes,
+             ignore_index,
+             logger=None):
+    """Semantic Segmentation  Evaluation.
 
      Evaluate the result of the Semantic Segmentation.
 
      Args:
-         gt_labels (list[torch.Tensor]): Ground truth labels.
-         seg_preds  (list[torch.Tensor]): Predtictions
-         label2cat (dict): Map from label to category.
-         logger (logging.Logger | str | None): The way to print the mAP
+        gt_labels (list[torch.Tensor]): Ground truth labels.
+        seg_preds  (list[torch.Tensor]): Predictions.
+        num_classes (int): Number of categories.
```

Comment:
Can num_classes be inferred by label2cat?

Reply:
Indeed.